packages/form/addon/components/cf-field/label.js

```diff
@@ -0,0 +1,20 @@
+import { getOwner } from "@ember/application";
+import Component from "@glimmer/component";
+
+/**
+ * Label component of the CfField
+ *
+ * @class CfFieldLabelComponent
+ */
+export default class CfFieldLabelComponent extends Component {
+  get config() {
+    return getOwner(this).resolveRegistration("config:environment");
+  }
+
+  get useMandatoryAsterisk() {
+    const { USE_MANDATORY_ASTERISK = false } =
+      this.config["ember-caluma"] || {};
+
+    return this.args?.useMandatoryAsterisk ?? USE_MANDATORY_ASTERISK;
```

Comment:
```suggestion
    return this.args.useMandatoryAsterisk ?? USE_MANDATORY_ASTERISK;
```

`this.args` is always an empty object in glimmer components even if not a single argument is passed.